src/data-types.js

```diff
@@ -87,6 +87,18 @@ export function getTypes(options) {
 		}
 	}
 
+	var int64 = {
+		unpack: function(buffer){
+			var value = buffer.readUInt64LE(buffer.pointer);
```

Comment:
I suggest you check out https://github.com/artemis-nerds/node-asbs-lib/blob/master/example/debugRawPacket.js (replace `playerShip` with `genericMesh`), capture a few generic mesh packets, and check that generic mesh packets can be parsed properly.